.github/workflows/publish-docker-images.yml

```diff
@@ -72,11 +72,12 @@ jobs:
 
       - name: Set Up Container Image Tags for zrok CLI Container
         env:
-          RELEASE_REPO: openziti/zrok
-          ZROK_VERSION: ${{ steps.semver.outputs.zrok_semver }}
+          ZROK_CONTAINER_IMAGE_REPO: ${{ vars.ZROK_CONTAINER_IMAGE_REPO || 'openziti/zrok' }}
```

Comment:
This corrects a flaw where a release in a fork will try unsuccessfully, or worse, successfully, to publish to the upstream's Docker Hub registry repo.